pkg/kubelet/cm/cpumanager/policy_options.go

```diff
@@ -109,6 +115,13 @@ func NewStaticPolicyOptions(policyOptions map[string]string) (StaticPolicyOption
 				return opts, fmt.Errorf("bad value for option %q: %w", name, err)
 			}
 			opts.AlignBySocket = optValue
+		case SpreadPhysicalCPUsPreferredOption:
+			optValue, err := strconv.ParseBool(value)
+			if err != nil {
+				return opts, fmt.Errorf("bad value for option %q: %w", name, err)
+			}
+			opts.SpreadPhysicalCPUsPreferredOption = optValue
+
```

Comment:
do we need some logic to fail loudly if both this option and `full-pcpus-only` are set at the same time, don't we?

Reply:
That's a good point. adding compatibility validation logic here sounds better.